virttest/libvirt_vm.py

```
@@ -2716,37 +2719,115 @@ def deactivate_nic(self, nic_index_or_name):
+                # Windows reboot patterns
+                r".*Microsoft Windows \[Version.*",
+                r".*Microsoft Corporation\. All rights reserved\.*",
+            ]
+            try:
+                if console.read_until_any_line_matches(
+                    reboot_patterns, timeout=timeout
+                ):
+                    LOG.debug("Reboot pattern detected in serial console")
+                    return True
+                LOG.debug(f"No reboot patterns detected within timeout {timeout} sec")
+                return False
```

Ruff (0.14.14)

Consider moving this statement to an `else` block

(TRY300)